### ISSUE TITLE
Add link to new List of SCs in the CVS summary

### ISSUE THREAD

**Issue by DavidMacDonald** [wet-boew/wet-boew-wpss#80]
I love the new summary columns adding up all SC at the end of the CSV summary

Consider adding a link to the WCAG Success criteria and the Help column, and adding the SC text in the TCID column

**Comment by irwink** [comment]
URL to WCAG technique/fail is provided in the rslt.csv file.
